p2p/dhtdiscovery/discovery.go

```diff
@@ -141,10 +147,51 @@ type Discovery struct {
 	bootstrapDuration   time.Duration
 	minPeers, highPeers int
 	backup, bootnodes   []peer.AddrInfo
+	pingPeers           []peer.ID
 }
 
 func (d *Discovery) Start() {
+	if len(d.pingPeers) != 0 {
```

Comment:
This is a quick temporary hack to get 2 peers to talk to each other for debugging. To be removed.